src/discord.cpp

```diff
@@ -27,6 +27,9 @@
 #include <string.h>
 #include "post_guard.h"
 
+// Uncomment this to provide some additional qDebug() output:
+#define DEBUG_DISCORD 1
```

Comment:
Should be commented out, right?

Reply:
Yep - for production usage...

Reply:
... have done this directly via the GitHub web interface.